### ISSUE TITLE
Lights stop responding to commands / z2m fills log with mqtt set/publish statements

### ISSUE THREAD

**Issue by stcbus** [Koenkk/zigbee2mqtt#11315]
### What happened?

I have noticed in the past few days that suddenly my zigbee network has become very unreliable. I had issues with my sengleds before, but this is different. it seems z2m gets in some weird state where it is DDOS' itself and publishing tons of mqtt messages and the lights aren't controllable. Sometimes z2m seems to "reset" on its own, but other times not.

I've attached a long long, but just look in the last 5 minutes or so of it if it's too long.

### What did you expect to happen?

Light turns on/off

### Zigbee2MQTT version

1.23.0

### Adapter firmware version

20211217

### Adapter

slaesh

### Debug log

[log.txt.gz](https://github.com/Koenkk/zigbee2mqtt/files/8028081/log.txt.gz)

**Comment by stcbus** [comment]
Here's another log, same thing happening. I shut off node red and tileboard just in case it was coming from them, but messages didn't stop going to the mqtt server until i shut down z2m, so the constant publishing is all z2m. it's odd. Hopefully someone has some insight!
[log2.txt.gz](https://github.com/Koenkk/zigbee2mqtt/files/8042160/log2.txt.gz)

**Comment by stcbus** [comment]
So I spent way too much time today troubleshooting this thing and I think it actually isn't an issue with z2m after all. My circadian flow seemingly randomly is duplicating the messages and sending numerous HA set commands which then correspond to z2m processing and getting overworked and crashing or giving buffer error messages.

This is super frustrating and no idea where the problem lies (HA, node red, HA node red modules) or if it's something I did but it is some change since updating :/

Sorry for the false report.
